bril-loop/README.md

```diff
@@ -0,0 +1,29 @@
+# Briloop
+* Briloop is a tool to convert bril programs into briloop format, converting unstructured control flow into structured control flow, using Ramsey's BeyondRelooper Algorithm. 
+
+## Usage
+* Pass a Bril JSON program to standard input, you'll obtain a Briloop JSON program in standard output. 
+```
+cat core/ackermann.bril | java -jar briloop.jar | ./run.sh
+```
+* `./briloop.sh` is an alias for the java command so you can also do: 
+```
+cat core/ackermann.bril | ./briloop | ./run.sh
+```
+
+## Requirements
+* Java 21
+
+## Installation
+* Run `./install.sh` (requires sudo) will create a symbolic link in /usr/local/bin
+
+## Building
+* Run `./setup.sh` to download jar dependencies. 
+* Run `./build.sh` to compile the project. 
+* Run `./run.sh` to run the transformation.
+* Run `./release.sh` to generate the release jar.
```

Comment:
Sorry if this is a silly question (I have not been in the JVM ecosystem for a long time), but is there a standard build tool (Gradle or Maven or [Amper](https://github.com/JetBrains/amper) or something) that we could use here to avoid custom shell scripting? A simple build setup could make this easier for people to use and maintain.